src/views/DocumentationTopic.vue

```diff
@@ -319,13 +350,22 @@ export default {
         this.toggleMobileSidenav();
       }
     },
+    openQuickNavigationModal() {
+      this.showQuickNavigationModal = true;
+    },
     toggleLargeSidenav(value = !this.sidenavHiddenOnLarge) {
       this.sidenavHiddenOnLarge = value;
       storage.set(NAVIGATOR_HIDDEN_ON_LARGE_KEY, value);
     },
     toggleMobileSidenav(value = !this.sidenavVisibleOnMobile) {
       this.sidenavVisibleOnMobile = value;
     },
+    onKeydown(event) {
```

Comment:
Maybe make this name relate to quickNavigation a bit more?

Reply:
This method gets called on any key down, so it doesn't have to be related to Quick navigation necessarily, I've been trying to come up with a different (but generic) name but haven't come up with any yet 🤔

Reply:
Resolved [dccfec7](https://github.com/apple/swift-docc-render/pull/396/commits/dccfec7956dd3d7b98fb0138097aaa05f31681e7)